spt.vcxproj

```diff
@@ -279,7 +282,9 @@
     </Bscmake>
     <PostBuildEvent>
       <Message />
-      <Command>copy "$(TargetPath)" "D:\Games\Source Unpack\portal\"</Command>
+      <Command>md "$(IntDir)..\..\builds"
+del "$(IntDir)..\..\builds\$(TargetFileName)"
+copy "$(TargetPath)" "$(IntDir)..\..\builds\$(TargetFileName)"</Command>
```

Comment:
where does this put the dll?

Reply:
ah in the builds/ folder

Reply:
Figured it was cleaner that way since you can use symlinks to get it anywhere else.